src/modules/iotjs_module_process.c

```diff
@@ -260,12 +260,17 @@ static void SetProcessEnv(jerry_value_t process) {
   iotjsenv = "";
 #endif
 
+  extra_module_path = getenv("IOTJS_EXTRA_MODULE_PATH");
```

Comment:
Could we use `IOTJS_MAGIC_STRING_IOTJS_EXTRA_MODULE_PATH_U`?

Reply:
I've changed it along with `"HOME"` and `"IOTJS_PATH"`. Thank you.